packages/pn-personafisica-webapp/src/pages/NotificationDetail.page.tsx

```diff
@@ -152,12 +152,6 @@ const NotificationDetail = () => {
   const legalFactDownloadUrl = useAppSelector(
     (state: RootState) => state.notificationState.legalFactDownloadUrl
   );
-  const legalFactDownloadRetryAfter = useAppSelector(
-    (state: RootState) => state.notificationState.legalFactDownloadRetryAfter
```

Comment:
remove the legalFactDownloadRetryAfter and legalFactDownloadAARRetryAfter from redux.
do it for all the webapp